javalin/src/test/java/io/javalin/TestStaticFiles.kt

```diff
@@ -34,15 +34,17 @@ class TestStaticFiles {
     private val defaultStaticResourceApp: Javalin by lazy { Javalin.create { it.addStaticFiles("/public") } } // classpath
     private val externalStaticResourceApp: Javalin by lazy { Javalin.create { it.addStaticFiles("src/test/external/", Location.EXTERNAL) } }
     private val multiLocationStaticResourceApp: Javalin by lazy {
-        Javalin.create { servlet ->
-            servlet.addStaticFiles("src/test/external/", Location.EXTERNAL)
-            servlet.addStaticFiles("/public/immutable")
-            servlet.addStaticFiles("/public/protected")
-            servlet.addStaticFiles("/public/subdir")
+        Javalin.create {
+            it.aliasCheckForStaticFiles = ContextHandler.AliasCheck { path, resource -> !path.endsWith(".txt") }
```

Comment:
I can't see tests with aliases which uses this `Javalin` instance? Is it safe to delete?

Reply:
Thanks, I just added these to all the instances to make sure the toggle didn't interfere with existing functionality, but forgot to remove them! Fixed now.